packages/scene-composer/src/components/panels/scene-settings/SceneTagSettingsEditor.tsx

```diff
@@ -0,0 +1,114 @@
+
+import useLifecycleLogging from '../../../logger/react-logger/hooks/useLifecycleLogging';
+import { useStore } from '../../../store';
+import { sceneComposerIdContext } from '../../../common/sceneComposerIdContext';
+import { IComponentSettingsMap, ITagSettings, KnownComponentType, KnownSceneProperty } from '../../../interfaces';
+import { componentSettingsSelector } from '../../../utils/componentSettingsUtils';
+import { Slider } from '../Slider';
+
+export const SceneTagSettingsEditor: React.FC = () => {
+  useLifecycleLogging('SceneTagSettingsEditor');
+
+  const sceneComposerId = useContext(sceneComposerIdContext);
+  const intl = useIntl();
+  const setSceneProperty = useStore(sceneComposerId)((state) => state.setSceneProperty);
+  const getSceneProperty = useStore(sceneComposerId)((state) => state.getSceneProperty);
+  const tagSettings: ITagSettings = useStore(sceneComposerId)((state) =>
+    componentSettingsSelector(state, KnownComponentType.Tag),
+  );
+  const [dirty, setDirty] = useState(false);
+  const [focusInput, setFocusInput] = useState(false);
+  const [focusSlider, setFocusSlider] = useState(false);
+
+  const [internalScale, setInternalScale] = useState(tagSettings.scale);
+
+  const updateSettings = useCallback(
+    (settingsPartial: Partial<ITagSettings>) => {
+      const newTagSettings: ITagSettings = {
+        ...tagSettings,
+        ...settingsPartial,
+      };
+      const newComponentSettings: IComponentSettingsMap = {
+        ...getSceneProperty(KnownSceneProperty.ComponentSettings),
+        [KnownComponentType.Tag]: newTagSettings,
+      };
+
+      setSceneProperty(KnownSceneProperty.ComponentSettings, newComponentSettings);
+    },
+    [tagSettings, getSceneProperty, setSceneProperty],
+  );
+
+  const onSliderChange = useCallback(
+    (event) => {
+      setInternalScale(Number(event.target.value));
+      setDirty(true);
+    },
+    [setInternalScale, setDirty],
+  );
+
+  const onInputBlur = useCallback(() => {
+    // If slider is getting focus, this makes sure to execute setFocusSlider(true) first to keep showing it.
+    const id = setTimeout(() => {
+      setFocusInput(false);
+    }, 1);
+    return () => {
+      clearTimeout(id);
+    };
+  }, [setFocusInput]);
+
+  const onInputChange = useCallback(
+    (event) => {
+      const value = event.detail.value;
+      setInternalScale(Number(value));
+      setDirty(true);
+    },
+    [setInternalScale, setDirty],
+  );
+
+  // Save scale changes to scene file
+  useEffect(() => {
+    if (dirty) {
+      updateSettings({ scale: internalScale });
+      setDirty(false);
+    }
+  }, [dirty]);
+
+  return (
+    <React.Fragment>
+      <FormField label={intl.formatMessage({ defaultMessage: 'Scale', description: 'Form Field label' })}>
+        <Grid gridDefinition={[{ colspan: 6 }, { colspan: 6 }]}>
+          <Input
+            value={String(internalScale)}
+            type='number'
```

Comment:
What will be the initial value of the input field?

Reply:
1 which is from the default settings I defined in constants